src/metabase/api/user.clj

```diff
@@ -160,9 +160,30 @@
   [clauses]
   (dissoc clauses :order-by :limit :offset))
 
+(defn- group-ids-for-manager
+  "Given a `user-id` return a list of group-ids of which the user is a group manager."
+  [user-id]
+  (map :group_id
+       (t2/query
+        {:select-distinct [:permissions_group_membership.group_id]
+         :from  [:permissions_group_membership]
+         :where [:and [:= :permissions_group_membership.user_id user-id]
+                 [:= :permissions_group_membership.is_group_manager true]
+                 [:not= :permissions_group_membership.group_id (:id (perms-group/all-users))]]})))
```

Comment:
nit: think it's slightly more idiomatic/readable to define an alias for the table name (e.g. `:from  [[:permissions_group_membership :pgm]]`) and use it in the rest of the query

Reply:
also nit: map is partially aligned